### PR TITLE
Update OpenAI Compatibility Docs with /v1/models/{model}

### PR DESCRIPTION
**Pull request by royjhan**

### CODE REVIEW

**Review by jmorganca — APPROVED** [review]
LGTM once we're ready to merge this

**Review by jmorganca — APPROVED** [review]

File: docs/openai.md
Comment (thinkverse) [inline docs/openai.md:79]: Not http://localhost:11434? 🤔

**Pull request merged**
